metrics/squad/README.md

```diff
@@ -0,0 +1,82 @@
+# Metric Card for SQuAD
+
+## Metric description
+This metric wraps the official scoring script for version 1 of the [Stanford Question Answering Dataset (SQuAD)](https://huggingface.co/datasets/squad). 
+
+SQuAD is a reading comprehension dataset, consisting of questions posed by crowdworkers on a set of Wikipedia articles, where the answer to every question is a segment of text, or span, from the corresponding reading passage, or the question might be unanswerable.
+
+## How to use 
+
+The metric takes two files or two lists of question-answers dictionaries as inputs : one with the predictions of the model and the other with the references to be compared to:
+
+```python
+from datasets import load_metric
+squad_metric = load_metric("squad")
+results = squad_metric.compute(predictions=predictions, references=references)
+```
+## Output values
+
+This metric outputs a dictionary with two values: the average exact match score and the average [F1 score](https://huggingface.co/metrics/f1).
+
+    {'exact_match': 100.0, 'f1': 100.0}
```

Comment:
Nit: maybe it's good to have all the code snippets and outputs as code blocks?

````suggestion
```
{'exact_match': 100.0, 'f1': 100.0}
```
````